src/app/query/bitcoin/transaction/use-check-utxos.ts

```diff
@@ -100,8 +100,9 @@ export function useCheckInscribedUtxos(blockTxAction?: () => void) {
 
       try {
         // no need to check for inscriptions on testnet
-        if (isTestnet) {
-          return false;
+        // TO_DO how to handle this in playwright mode?
+        if (!isTestnet) {
+          // return false;
```

Comment:
wonder how to better handle this... add some special IS_PLAYWRIGHT_ENV variable?

Reply:
I'm thinking of creating new env, like IS_PLAYWRIGHT_ENV, as we limit the app in test env, e.g.
```
  if (!IS_TEST_ENV)
    persistQueryClient({
      queryClient,
      persister: localStoragePersistor,
      buster: VERSION,
    });
```

Reply:
Playwright _is_ test env? Why do we need a new one?

Reply:
ah, it's unknown locally, on ci it's testing

Reply:
added